File: ISL.ReIdentification.WebApps.Portals/ISL.ReIdentification.Portals.Client/src/models/userAgreement/userAgreement.ts
export class UserAgreement {
    public id: string = "";
    public entraUserId: string = "";
    public agreementVersion: string = "";
    public agreementType: string = "";
    public agreementDate?: Date | undefined;
    public createdBy?: string;
    public createdDate?: Date | undefined;
    public updatedBy?: string;
    public updatedDate?: Date | undefined;
}

File: ISL.Reidentification.CoreTemp/Models/Foundations/UserAgreements/UserAgreement.cs
// ---------------------------------------------------------
// Copyright (c) North East London ICB. All rights reserved.
// ---------------------------------------------------------

using System;

namespace ISL.ReIdentification.Core.Models.Foundations.UserAgreements
{
    public class UserAgreement : IKey, IAudit
    {
        public Guid Id { get; set; }
        public string EntraUserId { get; set; }
        public string AgreementType { get; set; }
        public string AgreementVersion { get; set; }
        public DateTimeOffset AgreementDate { get; set; }
        public string CreatedBy { get; set; }
        public DateTimeOffset CreatedDate { get; set; }
        public string UpdatedBy { get; set; }
        public DateTimeOffset UpdatedDate { get; set; }
    }
}
